Tests/MMEEventsManagerTests.m

```diff
@@ -164,8 +164,6 @@ - (void)testTimerReachedWithEventQueued {
     
     [self.eventsManager pushEvent:locationEvent];
     XCTAssert(self.eventsManager.eventQueue.count > 0);
-    
-    XCTAssert(self.eventsManager.eventQueue.count == 0);
```

Comment:
Problem:
Event manager queue remains at zero

Regression:
https://github.com/mapbox/mapbox-events-ios/commit/876b4f2f5642ca28d5a066deaab775077d0cccf4#diff-b19b2c9fb2b4afec92d03b42c184b518L180

Why:
With the removal of timerManager, the events manager is no longer able to be manually flushed with trigger.

Change:
- Remove last comparison as it is no longer needed.